src/gsi_ncdiag/gsi_ncdiag.py

```diff
@@ -1593,6 +1593,16 @@ def toIODAobs(self, OutDir, ObsBias, QCVars, TestRefs, clobber=True):
                 outdata[gvname] = np.reshape(tmp, (nlocs, nchans))
                 VarDims[gvname] = ['Location', 'Channel']
 
+        if (TotalBias):
```

Comment:
Just noting that you may wish to do this for aircraft T as well as Ps, both of which have bias corrections applied.

Reply:
thanks Cory, think I will just add to all the class definitions